*** testsBoth/A-026-026-M-ok.out	Wed Oct 25 17:26:08 2023
--- testsBoth/A-026-026-M-ok.outhyp	Wed Oct 25 20:13:30 2023
***************
*** 29,36 ****
  2 - Procurar funções
  0 - Sair
  Escolha uma opção: 1
! Valor a procurar: 49
! 1;2|49
  Menu de Consultas
  1 - Procurar valores
  2 - Procurar funções
--- 29,40 ----
  2 - Procurar funções
  0 - Sair
  Escolha uma opção: 1
! Menu de Consultas
! 1 - Procurar valores
! 2 - Procurar funções
! 0 - Sair
! Escolha uma opção: 49
! Opção inválida!
  Menu de Consultas
  1 - Procurar valores
  2 - Procurar funções
