*** testsBoth/A-039-039-M-ok.out	2023-10-20 13:53:43.111842391 +0100
--- testsBoth/A-039-039-M-ok.outhyp	2023-10-20 23:37:22.867063387 +0100
***************
*** 17,23 ****
  0 - Sair
  Escolha uma opção: 1
  Especifique a gama (startline;startcol:endline;endcol): 4;2
! 4;2|#VALUE=SUB(2,1;3)
  Menu de Edição
  1 - Visualizar
  2 - Inserir
--- 17,23 ----
  0 - Sair
  Escolha uma opção: 1
  Especifique a gama (startline;startcol:endline;endcol): 4;2
! 4;2|#VALUE=SUB(1;3)
  Menu de Edição
  1 - Visualizar
  2 - Inserir
***************
*** 29,35 ****
  0 - Sair
  Escolha uma opção: 1
  Especifique a gama (startline;startcol:endline;endcol): 4;3
! 4;3|#VALUE=SUB(2,4;2)
  Menu de Edição
  1 - Visualizar
  2 - Inserir
--- 29,35 ----
  0 - Sair
  Escolha uma opção: 1
  Especifique a gama (startline;startcol:endline;endcol): 4;3
! 4;3|#VALUE=SUB(4;2)
  Menu de Edição
  1 - Visualizar
  2 - Inserir
