*** testsBoth/A-032-032-M-ok.out	Wed Oct 25 17:26:08 2023
--- testsBoth/A-032-032-M-ok.outhyp	Wed Oct 25 20:13:31 2023
***************
*** 11,21 ****
  2 - Procurar funções
  0 - Sair
  Escolha uma opção: 2
- Função a procurar: DIV
  Menu de Consultas
  1 - Procurar valores
  2 - Procurar funções
  0 - Sair
  Escolha uma opção: 0
  Menu Principal
  1 - Novo
--- 11,22 ----
  2 - Procurar funções
  0 - Sair
  Escolha uma opção: 2
  Menu de Consultas
  1 - Procurar valores
  2 - Procurar funções
  0 - Sair
+ Escolha uma opção: DIV
+ Número inválido!
  Escolha uma opção: 0
  Menu Principal
  1 - Novo
