*** testsBoth/A-053-053-M-ok.out	2023-10-20 13:53:43.115842359 +0100
--- testsBoth/A-053-053-M-ok.outhyp	2023-10-20 23:37:23.551059429 +0100
***************
*** 27,33 ****
  7 - Mostrar 'cut buffer'
  0 - Sair
  Escolha uma opção: 7
! 1;1|25
  Menu de Edição
  1 - Visualizar
  2 - Inserir
--- 27,33 ----
  7 - Mostrar 'cut buffer'
  0 - Sair
  Escolha uma opção: 7
! 2;1|25
  Menu de Edição
  1 - Visualizar
  2 - Inserir
