*** testsBoth/A-056-056-M-ok.out	2023-10-20 13:53:43.115842359 +0100
--- testsBoth/A-056-056-M-ok.outhyp	2023-10-20 13:59:27.141251201 +0100
***************
*** 29,50 ****
  0 - Sair
  Escolha uma opção: 3
  Especifique a gama (startline;startcol:endline;endcol): 1;1:55;1
- Copiar: Operação inválida: A gama '1;1:55;1' é inválida.
- Menu de Edição
- 1 - Visualizar
- 2 - Inserir
- 3 - Copiar
- 4 - Apagar
- 5 - Cortar
- 6 - Colar
- 7 - Mostrar 'cut buffer'
- 0 - Sair
- Escolha uma opção: 0
- Menu Principal
- 1 - Novo
- 2 - Abrir
- 3 - Guardar
- 4 - Menu de Edição
- 5 - Menu de Consultas
- 0 - Sair
- Escolha uma opção: 0
--- 29,31 ----
