*** testsBoth/A-066-066-M-ok.out	2023-10-20 13:53:43.119842327 +0100
--- testsBoth/A-066-066-M-ok.outhyp	2023-10-20 23:37:24.163055888 +0100
***************
*** 16,22 ****
  7 - Mostrar 'cut buffer'
  0 - Sair
  Escolha uma opção: 5
- Especifique a gama (startline;startcol:endline;endcol): 1;2
  Menu de Edição
  1 - Visualizar
  2 - Inserir
--- 16,21 ----
***************
*** 26,34 ****
  6 - Colar
  7 - Mostrar 'cut buffer'
  0 - Sair
  Escolha uma opção: 1
  Especifique a gama (startline;startcol:endline;endcol): 1;2
! 1;2|
  Menu de Edição
  1 - Visualizar
  2 - Inserir
--- 25,35 ----
  6 - Colar
  7 - Mostrar 'cut buffer'
  0 - Sair
+ Escolha uma opção: 1;2
+ Número inválido!
  Escolha uma opção: 1
  Especifique a gama (startline;startcol:endline;endcol): 1;2
! 1;2|49
  Menu de Edição
  1 - Visualizar
  2 - Inserir
***************
*** 39,60 ****
  7 - Mostrar 'cut buffer'
  0 - Sair
  Escolha uma opção: 7
- 1;1|49
- Menu de Edição
- 1 - Visualizar
- 2 - Inserir
- 3 - Copiar
- 4 - Apagar
- 5 - Cortar
- 6 - Colar
- 7 - Mostrar 'cut buffer'
- 0 - Sair
- Escolha uma opção: 0
- Menu Principal
- 1 - Novo
- 2 - Abrir
- 3 - Guardar
- 4 - Menu de Edição
- 5 - Menu de Consultas
- 0 - Sair
- Escolha uma opção: 0
--- 40,42 ----
