*** testsBoth/A-072-072-M-ok.out	2023-10-20 13:53:43.119842327 +0100
--- testsBoth/A-072-072-M-ok.outhyp	2023-10-20 13:59:27.741246947 +0100
***************
*** 27,33 ****
  7 - Mostrar 'cut buffer'
  0 - Sair
  Escolha uma opção: 6
- Especifique a gama (startline;startcol:endline;endcol): 5;1
  Menu de Edição
  1 - Visualizar
  2 - Inserir
--- 27,32 ----
***************
*** 37,47 ****
  6 - Colar
  7 - Mostrar 'cut buffer'
  0 - Sair
  Escolha uma opção: 1
  Especifique a gama (startline;startcol:endline;endcol): 5;1:5;3
! 5;1|15
! 5;2|49
! 5;3|15=1;1
  Menu de Edição
  1 - Visualizar
  2 - Inserir
--- 36,48 ----
  6 - Colar
  7 - Mostrar 'cut buffer'
  0 - Sair
+ Escolha uma opção: 5;1
+ Número inválido!
  Escolha uma opção: 1
  Especifique a gama (startline;startcol:endline;endcol): 5;1:5;3
! 5;1|
! 5;2|
! 5;3|
  Menu de Edição
  1 - Visualizar
  2 - Inserir
***************
*** 52,58 ****
  7 - Mostrar 'cut buffer'
  0 - Sair
  Escolha uma opção: 6
- Especifique a gama (startline;startcol:endline;endcol): 2;1
  Menu de Edição
  1 - Visualizar
  2 - Inserir
--- 53,58 ----
***************
*** 62,72 ****
  6 - Colar
  7 - Mostrar 'cut buffer'
  0 - Sair
  Escolha uma opção: 1
  Especifique a gama (startline;startcol:endline;endcol): 2;1:2;3
! 2;1|15
! 2;2|49
! 2;3|15=1;1
  Menu de Edição
  1 - Visualizar
  2 - Inserir
--- 62,74 ----
  6 - Colar
  7 - Mostrar 'cut buffer'
  0 - Sair
+ Escolha uma opção: 2;1
+ Número inválido!
  Escolha uma opção: 1
  Especifique a gama (startline;startcol:endline;endcol): 2;1:2;3
! 2;1|25
! 2;2|43
! 2;3|5=3;2
  Menu de Edição
  1 - Visualizar
  2 - Inserir
