*** testsBoth/A-083-083-M-ok.out	2023-10-20 13:53:43.123842295 +0100
--- testsBoth/A-083-083-M-ok.outhyp	2023-10-20 13:59:28.257243289 +0100
***************
*** 27,33 ****
  7 - Mostrar 'cut buffer'
  0 - Sair
  Escolha uma opção: 6
- Especifique a gama (startline;startcol:endline;endcol): 1;1:1;3
  Menu de Edição
  1 - Visualizar
  2 - Inserir
--- 27,32 ----
***************
*** 37,47 ****
  6 - Colar
  7 - Mostrar 'cut buffer'
  0 - Sair
  Escolha uma opção: 1
  Especifique a gama (startline;startcol:endline;endcol): 1;1:1;3
! 1;1|
! 1;2|
! 1;3|
  Menu de Edição
  1 - Visualizar
  2 - Inserir
--- 36,48 ----
  6 - Colar
  7 - Mostrar 'cut buffer'
  0 - Sair
+ Escolha uma opção: 1;1:1;3
+ Número inválido!
  Escolha uma opção: 1
  Especifique a gama (startline;startcol:endline;endcol): 1;1:1;3
! 1;1|15
! 1;2|49
! 1;3|15=1;1
  Menu de Edição
  1 - Visualizar
  2 - Inserir
***************
*** 52,60 ****
  7 - Mostrar 'cut buffer'
  0 - Sair
  Escolha uma opção: 7
! 1;1|
! 1;2|
! 1;3|
  Menu de Edição
  1 - Visualizar
  2 - Inserir
--- 53,61 ----
  7 - Mostrar 'cut buffer'
  0 - Sair
  Escolha uma opção: 7
! 5;1|
! 5;2|
! 5;3|
  Menu de Edição
  1 - Visualizar
  2 - Inserir
