*** testsBoth/A-090-090-M-ok.out	2023-10-20 13:53:43.123842295 +0100
--- testsBoth/A-090-090-M-ok.outhyp	2023-10-20 23:37:24.999051053 +0100
***************
*** 27,33 ****
  7 - Mostrar 'cut buffer'
  0 - Sair
  Escolha uma opção: 6
- Especifique a gama (startline;startcol:endline;endcol): 1;1
  Menu de Edição
  1 - Visualizar
  2 - Inserir
--- 27,32 ----
***************
*** 37,45 ****
  6 - Colar
  7 - Mostrar 'cut buffer'
  0 - Sair
  Escolha uma opção: 1
  Especifique a gama (startline;startcol:endline;endcol): 1;1
! 1;1|49
  Menu de Edição
  1 - Visualizar
  2 - Inserir
--- 36,46 ----
  6 - Colar
  7 - Mostrar 'cut buffer'
  0 - Sair
+ Escolha uma opção: 1;1
+ Número inválido!
  Escolha uma opção: 1
  Especifique a gama (startline;startcol:endline;endcol): 1;1
! 1;1|15
  Menu de Edição
  1 - Visualizar
  2 - Inserir
