*** testsBoth/A-091-091-M-ok.out	Wed Oct 25 17:26:08 2023
--- testsBoth/A-091-091-M-ok.outhyp	Wed Oct 25 20:13:33 2023
***************
*** 11,17 ****
  2 - Procurar funções
  0 - Sair
  Escolha uma opção: 2
! Função a procurar: 15
  Menu de Consultas
  1 - Procurar valores
  2 - Procurar funções
--- 11,22 ----
  2 - Procurar funções
  0 - Sair
  Escolha uma opção: 2
! Menu de Consultas
! 1 - Procurar valores
! 2 - Procurar funções
! 0 - Sair
! Escolha uma opção: 15
! Opção inválida!
  Menu de Consultas
  1 - Procurar valores
  2 - Procurar funções
