*** testsBoth/A-114-114-M-ok.out	Wed Oct 25 17:26:08 2023
--- testsBoth/A-114-114-M-ok.outhyp	Wed Oct 25 20:13:33 2023
***************
*** 19,25 ****
  Especifique a gama (startline;startcol:endline;endcol): 2;1:2;3
  2;1|25
  2;2|43
! 2;3|'123=COALESCE(1;1:1;4)
  Menu de Edição
  1 - Visualizar
  2 - Inserir
--- 19,25 ----
  Especifique a gama (startline;startcol:endline;endcol): 2;1:2;3
  2;1|25
  2;2|43
! 2;3|
  Menu de Edição
  1 - Visualizar
  2 - Inserir
