*** testsBoth/A-115-115-M-ok.out	Wed Oct 25 17:26:08 2023
--- testsBoth/A-115-115-M-ok.outhyp	Wed Oct 25 20:13:33 2023
***************
*** 17,23 ****
  0 - Sair
  Escolha uma opção: 1
  Especifique a gama (startline;startcol:endline;endcol): 2;3
! 2;3|'abc=COALESCE(1;1:1;4)
  Menu de Edição
  1 - Visualizar
  2 - Inserir
--- 17,23 ----
  0 - Sair
  Escolha uma opção: 1
  Especifique a gama (startline;startcol:endline;endcol): 2;3
! 2;3|
  Menu de Edição
  1 - Visualizar
  2 - Inserir
