*** testsBoth/A-122-122-M-ok.out	2023-10-20 13:53:43.127842262 +0100
--- testsBoth/A-122-122-M-ok.outhyp	2023-10-20 23:37:25.575047721 +0100
***************
*** 11,30 ****
  2 - Procurar funções
  0 - Sair
  Escolha uma opção: 1
- Valor a procurar: '123
- 1;1|'123
- 2;4|'123=COALESCE(1;1:1;2)
  Menu de Consultas
  1 - Procurar valores
  2 - Procurar funções
  0 - Sair
  Escolha uma opção: 1
- Valor a procurar: '123456
- 2;3|'123456=CONCAT(1;1:1;2)
  Menu de Consultas
  1 - Procurar valores
  2 - Procurar funções
  0 - Sair
  Escolha uma opção: 0
  Menu Principal
  1 - Novo
--- 11,29 ----
  2 - Procurar funções
  0 - Sair
  Escolha uma opção: 1
  Menu de Consultas
  1 - Procurar valores
  2 - Procurar funções
  0 - Sair
+ Escolha uma opção: '123
+ Número inválido!
  Escolha uma opção: 1
  Menu de Consultas
  1 - Procurar valores
  2 - Procurar funções
  0 - Sair
+ Escolha uma opção: '123456
+ Número inválido!
  Escolha uma opção: 0
  Menu Principal
  1 - Novo
