*** testsBoth/A-120-120-M-ok.out	2023-10-20 13:53:43.127842262 +0100
--- testsBoth/A-120-120-M-ok.outhyp	2023-10-20 23:37:25.515048067 +0100
***************
*** 88,94 ****
  0 - Sair
  Escolha uma opção: 1
  Especifique a gama (startline;startcol:endline;endcol): 2;3
! 2;3|'789=COALESCE(1;1:1;3)
  Menu de Edição
  1 - Visualizar
  2 - Inserir
--- 88,94 ----
  0 - Sair
  Escolha uma opção: 1
  Especifique a gama (startline;startcol:endline;endcol): 2;3
! 2;3|
  Menu de Edição
  1 - Visualizar
  2 - Inserir
***************
*** 100,106 ****
  0 - Sair
  Escolha uma opção: 1
  Especifique a gama (startline;startcol:endline;endcol): 2;4
! 2;4|1075=PRODUCT(2;1:2;2)
  Menu de Edição
  1 - Visualizar
  2 - Inserir
--- 100,106 ----
  0 - Sair
  Escolha uma opção: 1
  Especifique a gama (startline;startcol:endline;endcol): 2;4
! 2;4|
  Menu de Edição
  1 - Visualizar
  2 - Inserir
